NewSpring/crete
# chore: Improves Developer Experience

## DESCRIPTION

### What does this PR do, or why is it needed?

Makes it so the server works with the latest Node and we don't have to have `nvm` installed. Also makes it so that the `PORT` env variable isn't required.

### How do I test this PR?

Switch to the latest node (I think 14 something) and delete the `PORT` variable and start the server and run tests. Should have no issues.

## TODO


## REVIEW


---

> The purpose of a PR Review is to _improve the quality of the software._
